Alignment/OfflineValidation/python/TkAlAllInOneTool/primaryVertexValidation.py

```diff
@@ -28,9 +28,9 @@ def __init__(self, valName, alignment, config):
 
         if self.general["pvvalidationreference"].startswith("/store"):
             self.general["pvvalidationreference"] = "root://eoscms//eos/cms" + self.general["pvvalidationreference"]
-        if self.NJobs > 1:
-            raise AllInOneError("Parallel jobs not implemented for the PrimaryVertex validation!\n"
-                                "Please set parallelJobs = 1.")
+#        if self.NJobs > 1:
```

Comment:
@cardinia why keep the commented out lines?

Reply:
@mmusich , they can be removed without problems. I initially thought that keeping them could be useful when working on the Parallel Validation code itself to loosen some of the current limitations (number of jobs to be sent in parallel or the need to specify the maximum number of events). But that was just a thought, their usefulness is basically none, I can remove those lines without problems.